docs/tutorials/Monge_Gap.ipynb

```diff
@@ -1,5 +1,15 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.        model = models.models.MLP(
</pre><p>maybe import models to avoid <code>models.models</code> ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/467/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Now it should be <code>from ott.neural import models; model = models.MLP</code> </p><p> </p>